src/main/scala/Monday_Basic.scala

```diff
@@ -0,0 +1,150 @@
+object Monday_Basic extends App {
+
+  //Basic
+  //Output "Hello World via the println() method"
+  println("Hello World")
+
+  //Store "Hello World!" in a variable and then output it
+  var output = "Hello World"
+  println(output)
+
+  //Create a method that accepts a string as a parameter and then outputs that string
+  def output(word: String) = {
+    println(word)
+  }
+
+  output("William")
+
+  //Create a method that returns a string, in this case “Hello World!”, which you then use to output
+  def returnOutput(): String = {
+    var name = "Manchester"
+    return name
+  }
+
+  println(returnOutput)
+
+  //Crate a method that does not signify a return type and outputs whatever is given to it then
+  //test it by passing the following data types, Double, String, Integer, Boolean
+  def squareOf(x: Int) = x * x
+
+  println(squareOf(12))
+
+  //Create a method that accepts two parameters, one being a string and one being an integer.
+  //Your method should return the amount of characters from the end of the string that the integer specifies
+
+  //def lastLetters(x:String,y:Int){
+
+  def stringRemover(word: String, x: Int) {
+    var output = word.substring(word.length - x)
+    println(output)
```

Comment:
you're not reassigning value to output, method body could be simplified to:

println(word.substring(word.length - x))